databooks/logging.py

```diff
@@ -1,12 +1,14 @@
 """Logging helper functions."""
 
 import logging
+import os
 
 from rich.logging import RichHandler
 
 
-def get_logger(name: str, level: str = "INFO") -> logging.Logger:
+def get_logger(name: str) -> logging.Logger:
     """Get logger with rich configuration."""
+    level = os.getenv("LOG_LEVEL", logging.INFO)
```

Comment:
Don't we want to be able to overwrite this with a config file/cli arg? We can definitely do this in another PR as well, just mentioning it

Reply:
Actually this came from [another PR](https://github.com/datarootsio/databooks/pull/4) but it got lost somewhere (probably when I was rebasing or something).
In the CLI, you can pass the `--verbose` argument that will pass it to downstream functions. When the argument is passed, we call the [`set_verbose(logger)`](https://github.com/datarootsio/databooks/pull/18/files/59bcacc8c1c751f546474d6ab78f919d9f440f5b#diff-45ceef528db0c2cc8bdd915a8031106ea55c828fd2d435c3bb47ad69774e3abfR21)
And since every CLI argument is configurable, then you can also configure it in the CLI 🚀 .

The thing I like about this approach is that we can still increase the amount of logs, cause when passing `--verbose` you only "reach" the functions that take that as an input. Whereas if you really want to debug stuff you can set all loggers everywhere. This is what I tried explaining [here](https://github.com/datarootsio/databooks/pull/18/files/59bcacc8c1c751f546474d6ab78f919d9f440f5b#diff-45ceef528db0c2cc8bdd915a8031106ea55c828fd2d435c3bb47ad69774e3abfR21)

Reply:
Ok cool, that makes sense!